collect_app/src/main/java/org/odk/collect/android/architecture/rx/RxMVVMViewModel.java

```diff
@@ -1,6 +1,7 @@
 package org.odk.collect.android.architecture.rx;
 
 
+import android.os.Bundle;
```

Comment:
Hi! I claim that initialisms (like MVVM) embedded in type names should have only the first letter capitalized. And here’s my appeal to authority:

[BlochMenOfTaste.mp3.zip](https://github.com/opendatakit/collect/files/1630087/BlochMenOfTaste.mp3.zip)

Reply:
Yep, you're right! I've been Swifting and got my naming conventions mixed up.